### PR TITLE
Minor fixes for OKD GSG

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

**Comment by errordeveloper** [comment]
@twpayne since you are testing this guide, could you please review this very minor change?

**Review by twpayne — APPROVED** [review]

File: Documentation/gettingstarted/k8s-install-openshift-okd.rst [hunk 1]
```diff
@@ -151,7 +152,10 @@ Next, obtain Cilium manifest from ``cilium/cilium-olm`` repository and copy to `
 
    .. code-block:: shell-session
 
-       sed -i 's|image:\ registry.connect.redhat.com/isovalent/|image:\ quay.io/cilium/|g' "${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00002-cilium-olm-deployment.yaml"``
+       sed -i 's|image:\ registry.connect.redhat.com/isovalent/|image:\ quay.io/cilium/|g' \
+         "${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00002-cilium-olm-deployment.yaml" \
+         ${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00014-cilium.*-clusterserviceversion.yaml
```
Comment (qmonnet) [inline Documentation/gettingstarted/k8s-install-openshift-okd.rst:157]:
```suggestion
         "${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00014-cilium.*-clusterserviceversion.yaml"
```

Double-quotes for consistency with the line above?

**Review by qmonnet — APPROVED** [review]
Looks good, with one nit

File: Documentation/gettingstarted/k8s-install-openshift-okd.rst (hunk 1 above)
Comment (errordeveloper) [reply Documentation/gettingstarted/k8s-install-openshift-okd.rst:157]: This was intentional because there is a `*`...

**Pull request merged**